libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -91,7 +91,14 @@ const AP_Param::GroupInfo AP_Camera::var_info[] = {
     // @Values: 0:TriggerLow,1:TriggerHigh
     // @User: Standard
     AP_GROUPINFO("FEEDBACK_POL",  9, AP_Camera, _feedback_polarity, 1),
-    
+
+    // @Param: FEEDBACK_SOURCE
+    // @DisplayName: Camera feedback location source
+    // @Description lalalala
+    // @Values: 0:Normal,1:GPS1,2:GPS2
```

Comment:
`@DisplayName: An option where you can chose the camera feedback location provider.`

Is this better?